packages/react-sdk/src/pubsub/create.ts

```diff
@@ -66,5 +66,19 @@ export const createPubsub = <PublishMap>() => {
         };
       }, [type, onAction]);
     },
+
+    useSubscribeAll(
```

Comment:
do we really need this function or could we just subscribe using asterix?

Reply:
Just tricky with TS, I wouldn't spend time on trying to solve it